### PR TITLE
Added an appdata.xml file for Linux software gallery integration

### PR DESCRIPTION
**Pull request by Mailaender**
See https://people.freedesktop.org/~hughsient/appdata/ for more information.

### CODE REVIEW

**Comment by Mailaender** [comment]
See https://build.opensuse.org/request/show/433055 for an integration test. I am not really sure what's wrong with the rpmlint checker as it crashes. The file seems to otherwise validate just fine.

**Comment by elbenfreund** [comment]
Thank you for your feedback, it is much appreciated.

Project hamster and its various sub-components is currently undergoing some major changes. We prepare the introduction of a rewritten codebase for most of the underlying functionality. A direct consequence of this is that it is unlikely that any open/new bugs within the current/old codebase will be fixed (unless someone steps up and offers to do so) as most resources currently available will be invested in making the rewrite prime time ready.

If you are still interested in working on the new codebase (repositories: hamster-lib/cli/gtk/dbus) we would be most thrilled. Please feel free to either open a new issue with the relevant repository and/or join the discussion on the [mailinglist](http://lists.projecthamster.org/listinfo.cgi/hamster-dev-projecthamster.org).
For more context please see our [website](http://projecthamster.org).

Thanks for your interest and support! Eric.

File: data/wscript_build [hunk 1]
```diff
@@ -12,6 +12,7 @@ bld.install_files('${DATADIR}/icons/hicolor/32x32/apps',   'art/32x32/hamster-ti
 bld.install_files('${DATADIR}/icons/hicolor/48x48/apps',   'art/scalable/hamster-time-tracker.png')
 bld.install_files('${DATADIR}/icons/hicolor/scalable/apps','art/scalable/hamster-time-tracker.svg')
 
+bld.install_files('${DATADIR}/appdata/hamster-time-tracker.appdata.xml', 'hamster-time-tracker.appdata.xml')
```
Comment (DimStar77) [inline data/wscript_build]: Destination must not contain the file name - only the destination directory.

Comment (Mailaender) [inline data/wscript_build]: Fixed.

**Comment by Mailaender** [comment]
https://build.opensuse.org/request/show/433873 now all checks succeed.

**Comment by ederag** [comment]
Thanks for your contributions !
We definitely need this `appdata.xml`, but the interface changed a lot meanwhile.
Screenshots linked in this `appdata.xml` are for the "hamster-applet" (before 2.0) version.
So the best course of action might be to rebase this PR on the [hamster_stable](https://github.com/projecthamster/hamster/tree/hamster_stable) branch,
so that I can merge it there.

Then a separate PR would be best, based on the `master` branch, with updated screenshots.
It should be possible to have the main `screenshot.png` in the root directory,
and other screenshots in a `data/screenshots/` directory,
all tracked by git (png screenshots should be small enough for that).
Later on, it will be possible to get [links for a specific release version](https://help.github.com/articles/getting-permanent-links-to-files/),
and automate the process with [bump2version](https://github.com/c4urself/bump2version) for instance.

**Comment by ederag** [comment]
Well, [changing the base branch myself](https://help.github.com/en/articles/changing-the-base-branch-of-a-pull-request) turned out to be a mess. :disappointed_relieved: 
Revert. :relieved:

**Comment by ederag** [comment]
This work is being updated in PR #441. Closing, thanks for your contribution !

**Comment by ederag** [comment]
@Mailaender closed, but the proposition to rebase on the [hamster-stable](https://github.com/projecthamster/hamster/tree/hamster_stable) branch still holds, 
if you want to get it merged there. Thanks again !

**Pull request closed without merging**